content/academy/professional_scraping/solutions/actor_building.md

```diff
@@ -513,6 +500,46 @@ We've also replaced the default `title` and `description` with our own custom on
 
 Let's try it out now! Input **iphone** into the box labeled **keyword**, click **Start**, and see the sparks fly!
 
+## [](#quiz-answers) Quiz answers 📝
+
+- When using Puppeteer or Playwright, how can you still use jQuery with the SDK?
+
+There are two ways. You can either use the [injectJQuery](https://sdk.apify.com/docs/api/puppeteer#puppeteerinjectjquerypage) utility function which will enable you to use jQuery inside of `page.evalute()`, or you can use Cheerio to load the page's content like this:
+
+```JavaScript
+const $ = cheerio.load(await page.content());
+```
+
+Which will allow you to use jQuery syntax outside of Puppeteer/Playwright page evaluation functions.
+
+- What is the main difference between Cheerio and jQuery?
+
+Though they share the same syntax, and therefore look quite similar, Cheerio and jQuery are designed for different purposes. jQuery is a frontend library for manipulating the DOM, making AJAX calls, etc. while Cheerio is a backend library exclusively for Node.js built for the purpose of intuitively parsing HTML on the server-side.
+
+- In which situations would you use CheerioCrawler? What are its limitations?
+
+One should use CheerioCrawler when scraping any non-dynamic content. For scraping any content that doesn't require the loading of JavaScript in order to receive all of the data (such as with server-side rendered HTML pages and APIs), CheerioCrawler should be used. It is limited though, as it can only make static requests. This means that if a piece of data is loaded using JavaScript from an API call that the page makes, CheerioCrawler will never see that piece of data.
+
+> Learn more about dynamic pages in our [dynamic pages]({{@link dynamic_pages_and_spas/js_rendering_and_dynamic_pages.md}}) lesson.
+
+Additionally, if the job being done requires some sort of interaction with the page, PlaywrightCrawler/PuppeteerCrawler should be used, as CheerioCrawler runs out of the context of the browser.
+
+- Using Puppeteer, how can you extract data from a page without using jQuery/Cheerio?
+
+You can use functions such as `page.evaluate()` and `page.$$eval()` to run data collection code in the context of the browser and return it so that it can be used back in the Node.js context.
```

Comment:
Let's also add `document.querySelector` example with evaluate but mention that JQuery is preferable.